### PR TITLE
Additional output for `pg:backups` commands

### PR DESCRIPTION
**Pull request by chadbailey59**
This PR ensures we're outputting as much info as we can for `pg:backups` and `pg:backups info` commands.

### CODE REVIEW

**Comment by chadbailey59** [comment]
Also, two bug fixes:

1. Restores that came from `htcat` wouldn't show in restores.
2. If an app had no backups but did have restores/copies, they wouldn't show because 'no backups' was using the `error` helper, which (helpfully) exits.

**Comment by neovintage** [comment]
@chadbailey59 I tested out the code.  I have a couple of comments. 

* when you do pg:copy of databases across applications,  there's no way of knowing that it happened.  You just see two different colors.  
* The output for copies can be exceptionally long.  Is there a way to limit that to maybe the last 10?

I have screenshots if you need them.

File: lib/heroku/command/pg_backups.rb [hunk 1]
```diff
@@ -399,7 +422,7 @@ def poll_transfer(action, transfer_id)
       redisplay <<-EOF
 An error occurred and your backup did not finish.
 
-Please run `heroku logs --ps pg-backups` for details.
+Please run `heroku pg:backups info #{transfer_name(backup)}` for details.
```
Comment (uhoh-itsmaciek) [inline lib/heroku/command/pg_backups.rb]: Yes thank you. I've been meaning to do this for forever and keep forgetting.

File: .ruby-version [hunk 2]
```diff
@@ -0,0 +1 @@
+2.2.0
```
Comment (uhoh-itsmaciek) [inline .ruby-version]: This should probably be separate or just skipped.

**Comment by uhoh-itsmaciek** [comment]
Looks great. I haven't tried it, but I think your changes look pretty straightforward, so with you and Rimas giving it a shot (and the specs) I'm sure it's fine. Let me know if you want me to run through something in particular.

I'm neutral on the log output @neovintage brought up--that can certainly be a lot of output, but you're generally only looking when you want to know so maybe that's fine? I don't follow the cross-app copy issue. Maybe the screenshot would help.

**Comment by neovintage** [comment]
The setup is I have two apps, neovintage and pgbackups-rims.  I copied gold
from neovintage to Blue on pgbackups-rims.  When I look at this output on
pgbackups-rims, I only see GOLD to BLUE.  I don't have context that it came
from another app.

**Comment by neovintage** [comment]
Screenshot
screen shot 2015-04-27 at 9 14 45 pm

**Comment by uhoh-itsmaciek** [comment]
Got it. I think I know how to fix that, but maybe we should address that independently of these changes so these can go out more quickly...

**Comment by chadbailey59** [comment]
I removed the ruby-version file. I also limited the restore and copy outputs to the 20 most recent.

I absolutely think @neovintage 's concern about cross-app restores is valid. I noticed it too when I was testing, but I'm trying to keep this as limited in scope as possible to get the bugfixes out.

**Comment by neovintage** [comment]
Is that 20 total across both or each?  I would suggest limiting to last
10 in each.

What about the web interface to all of this?

**Comment by chadbailey59** [comment]
I just pushed a commit limiting to the last 10 operations each. It'll show
10 most recent restores and 10 most recent pg:copy operations.

As far as the web, I'm not sure.

**Comment by neovintage** [comment]
Thanks for the fix.  We can push this out now.  We'll have to schedule some time to get PHC fixed to include cross app copies and restores.  For future reference, we need to make sure that if we make changes to CLI we evaluate if a change is necessary to PHC.  

/cc @chadbailey59 @uhoh-itsmaciek

**Pull request merged**